src/BreedSelect.jsx

```diff
@@ -0,0 +1,22 @@
+const BreedSelect = (props) => {
```

Comment:
First way: in first line on stary component make something like that const { arg1, arg2 } = props 
Second way: const BreedSelect = ({ arg1, arg2 }) => {